composer.json

```diff
@@ -20,10 +20,16 @@
     "require-dev": {
         "phpunit/phpunit": "4.8.*",
         "squizlabs/php_codesniffer": "2.*",
-        "google/cloud-trace": "^0.3",
+        "google/cloud": "dev-trace-v2",
```

Comment:
Does it make sense to file an issue to change it back to tagged release?

Reply:
Will do. The google-cloud-php PR is approved and will merge when we release opencensus to packagist.